lib/utils/checkAgainstRule.js

```diff
@@ -38,6 +38,7 @@ module.exports = function (options, callback) {
 		return;
 	}
 
+	// @ts-expect-error - this error should not occur with PostCSS 8
 	const tmpPostcssResult = new Result();
```

Comment:
[PostCSS 8](https://github.com/postcss/postcss/releases/tag/8.0.0) has a first-class TypeScript support. Our migration is progressing on #4942.

Related:

https://github.com/stylelint/stylelint/blob/6b06b36c37820513d08c98e3a06d88551e7cc9eb/lib/utils/checkAgainstRule.js#L4

https://github.com/stylelint/stylelint/blob/6b06b36c37820513d08c98e3a06d88551e7cc9eb/types/postcss/index.d.ts#L47-L51